tagbotgitlab/tagbot.py

```diff
@@ -83,6 +83,14 @@ def handle_open(payload):
         timeout += 1
         mr = p.mergerequests.get(mr_id, lazy=False)
 
+    # Accepting the MR while the merge_status is "checking" seems to result in a
+    # 406 error.
+    # To work around this, poll until the merge_status is no longer "checking".
+    # See https://gitlab.com/gitlab-org/gitlab/-/issues/196962
+    while mr.merge_status == "checking":
+        print(f"Polling for merge_status: {mr.merge_status}")
```

Comment:
These 2 lines aren't actually being tested. you'll need to set one of the mocked MR's as `checking` so it goes into the loop. I'm not sure if any other test does this but you'll need the first check to be `checking` and then the second check be `can_be_merged`